tests/server.test.js

```diff
@@ -25,3 +26,110 @@ describe('Person Route', () => {
     expect(response.statusCode).toBe(500);
   });
 });
+
+
+describe('Artist Routes', () => {
+  beforeEach(() => sequelize.sync());
+  afterEach(() => sequelize.drop());
+
+  it('Creates an artist', async () => {
+    const response = await request.post('/artist').send({
+      name: 'Giveon',
+      single: 'For Tonight',
```

Comment:
After this line, add 2 more 'post' as this will serve as your 'database' for example:
```
const response2 = await request.post('/artist').send({
      name: 'JVKE',
      single: 'Golden Hour',
    });
    const response3 = await request.post('/artist').send({
      name: 'The Weeknd',
      single: 'Out of Time',
    });
  const response4 = await request.post('/artist').send({
      name: 'Doja Cat',
      single: 'Meow',
    });
```